distribution/BUILD

```diff
@@ -73,3 +75,49 @@ sh_binary(
         "//tools/distribution:verify",
     ],
 )
+
+label_flag(
+    name = "x64-packages",
+    build_setting_default = "//distribution:custom/x64/packages.x64.tar.gz",
+)
+
+label_flag(
+    name = "arm64-packages",
+    build_setting_default = "//distribution:custom/arm64/packages.arm64.tar.gz",
+)
+
+label_flag(
+    name = "x64-release",
+    build_setting_default = "//distribution:custom/x64/bin/release.tar.zst",
+)
+
+label_flag(
+    name = "arm64-release",
+    build_setting_default = "//distribution:custom/arm64/bin/release.tar.zst",
+)
+
+genrule(
+    name = "signed",
+    outs = ["release.signed.tar.zst"],
+    cmd = """
+    # Sign the packages
+    $(location //tools/distribution:sign) \
+        "deb.x64:$(location :x64-packages)" \
+        "deb.arm64:$(location :arm64-packages)" \
```

Comment:
currently these packages are added without mapping the deb/changes files into the `bin/` folder which is what we want for release